recipes/fast-cdr/all/conanfile.py

```diff
@@ -59,23 +59,12 @@ def _source_subfolder(self):
 
     def _create_cmake_module_alias_target(self):
         content = ""
-        if self.options.shared and self.settings.os == "Windows":
-            # alias plus special properties which are required
-            content += textwrap.dedent("""\
-                if(TARGET fastcdr::fastcdr AND NOT TARGET fastcdr)
-                    add_library(fastcdr INTERFACE IMPORTED)
-                    set_property(TARGET fastcdr PROPERTY INTERFACE_LINK_LIBRARIES fastcdr::fastcdr)
-                    set_property(TARGET fastcdr PROPERTY INTERFACE_COMPILE_DEFINITIONS FASTCDR_DYN_LINK)
-                endif()
-                set_property(TARGET fastcdr::fastcdr PROPERTY INTERFACE_COMPILE_DEFINITIONS FASTCDR_DYN_LINK)
-            """)
-        else:
-            content += textwrap.dedent("""\
-                if(TARGET fastcdr::fastcdr AND NOT TARGET fastcdr)
-                    add_library(fastcdr INTERFACE IMPORTED)
-                    set_property(TARGET fastcdr PROPERTY INTERFACE_LINK_LIBRARIES fastcdr::fastcdr)
-                endif()
-            """)
+        content += textwrap.dedent("""\
+            if(TARGET fastcdr::fastcdr AND NOT TARGET fastcdr)
+                add_library(fastcdr INTERFACE IMPORTED)
+                set_property(TARGET fastcdr PROPERTY INTERFACE_LINK_LIBRARIES fastcdr::fastcdr)
```

Comment:
https://github.com/conan-io/conan-center-index/blob/e2964fbef5828621a7c579991717003076f7818b/recipes/parallel-hashmap/all/conanfile.py#L51

Usually we use a format, just a comment... this is perfectly correct